frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/strings/strings.ts

```diff
@@ -26,10 +26,11 @@ export const TabTitleResolver = {
 };
 
 export const BASE_IMAGE_AND_PVC_SHORT = '(Source available)';
+export const BASE_IMAGE_AND_PVC_UPLOADING_SHORT = '(Source uploading)';
 export const NO_BASE_IMAGE_SHORT = '';
 export const NO_BASE_IMAGE_AND_NO_PVC_SHORT = '';
 export const BASE_IMAGE_AND_PVC_MESSAGE = '';
-export const NO_BASE_IMAGE_MESSAGE =
-  'The disk image defined for this Operating System is not available, a custom boot source must be defined manually';
 export const NO_BASE_IMAGE_AND_NO_PVC_MESSAGE =
   'The Operating System Template is missing disk image definitions, a custom boot source must be defined manually';
+export const BASE_IMAGE_UPLOADING_MESSAGE =
+  'The Source for this Operating System is being uploaded and cannot be used';
```

Comment:
We should say
`The upload process for [Operating system name] must complete before it can be cloned`

Reply:
determined that using the OS name is not very readable so marking this as resolved.